tests/interfaces/legacy_devices_integration/test_autograd_qnode_legacy.py

```diff
@@ -1705,8 +1705,8 @@ def circuit(data, weights, coeffs):
 
         # test second-order derivatives
         if diff_method == "parameter-shift" and max_diff == 2:
-            with pytest.warns(UserWarning, match=r"Output seems independent of input."):
```

Comment:
No more warning?

Reply:
Nope. Is that a problem?

Reply:
This just seems rather suspicious.

Reply:
something to do with running `hamiltonian_expand`in the "inner execute" instead of outside the ml boundary. I will further investigate.

Especially since we actually get:
```
>>> qml.jacobian(qml.grad(circuit, argnum=2), argnum=2)(d, w, c)
<autograd.numpy.numpy_boxes.ArrayBox at 0x30556a600>
```

which is probably not an answer we want to return.